README.md

```diff
@@ -89,7 +93,13 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD is an entity relationship diagram.  These diagrams are used to show the relationship between (and contents of) entities, which are generally classes.  These diagrams map out the relationships that will exist between two or more related classes and so help to demonstrate which methods will be required in order to represent the diagram inside code.
+
```

Comment:
I would suggest saying `model` instead of `class` and you're golden. We tend to use ERD's to map the relationship between the models in our data, which we define as a class.